web/docs/get-started.md

```diff
@@ -205,6 +205,14 @@ storage.
   </TabItem>
 </Tabs>
 
+:::caution Unsafe Pipelines
+Some operators are inherently unsafe because of their side effects, e.g.,
+reading a file, acquiring packets from the network, or using a third-party
+library. We do not allow these operators by default in managed pipelines. You
```

Comment:
I don't think we ever use the term "managed pipeline" outside of code.

Reply:
I'm open to suggestions.

We need to differentiate them from pipelines created with the `tenzir` binary. Something along the lines of "pipelines managed by a node" or "pipelines created through the API".

Reply:
The restriction applies to all _local_ operators that are run remotely instead. This includes:
- All _local_ operators that are started through the API.
- All _local_ operators that are explicitly switched to run remotely with the `remote` modifier.

Not sure on the wording. I think we should start marking the default operator location visibly in the operator docs so that this is easier to understand.

Reply:
I tweaked the description. Resolving, since we paired on it.